### PR TITLE
MM-24440: Don't allow clearing the site url

### PR DESCRIPTION
**Pull request by ashishbhate**
#### Summary

- If the site URL is set, don't allow clearing it.

#### Ticket Link

- https://mattermost.atlassian.net/browse/MM-24440

### CODE REVIEW

**Comment by grundleborg** [comment]
@ashishbhate is there a webapp PR to go with this?

**Comment by ashishbhate** [comment]
@grundleborg Since the system console calls the API, I assumed this change was to be implemented on the API side. Attempting to clear the site URL in the in the system console with this change included displays "Can't clear Site URL." in red at the bottom.

**Comment by grundleborg** [comment]
Ah makes sense. @wiersgallak is this sufficient or should there also be client side validation in the system console UI?

**Comment by wiersgallak** [comment]
@thefactremains Can you review the solution on this, since I know you guys were looking at validation rules on the System Console?

**Comment by ashishbhate** [comment]
@wiersgallak gentle bump for PM review approval.

File: i18n/en.json [hunk 1]
```diff
@@ -1114,6 +1114,10 @@
     "id": "api.config.client.old_format.app_error",
     "translation": "New format for the client configuration is not supported yet. Please specify format=old in the query string."
   },
+  {
+    "id": "api.config.update_config.clear_siteurl.app_error",
+    "translation": "Can't clear Site URL."
```
Comment (wiersgallak) [inline i18n/en.json:1119]: @ashishbhate Can we update this to "Site URL cannot be cleared."

Comment (ashishbhate) [reply i18n/en.json]: @wiersgallak done

**Pull request merged**